addons/binding/org.openhab.binding.tivo/ESH-INF/thing/thing-types.xml

```diff
@@ -0,0 +1,152 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
Done (I think).  Used Source > Cleanup Document in Eclipse.  If not what you think is wrong can you clarify as I'm missing the issue.